app.rb

```diff
@@ -0,0 +1,116 @@
+require 'student'
+require 'teacher'
+require 'book'
+require 'rental'
```

Comment:
- [ ] Please consider using `require_relative` instead of `require` to load your local modules. Currently, I'm getting this error:

![image](https://github.com/Cisco-the-wayword/OOP-SchoolLibrary/assets/77069011/c5b2e506-817e-4bc4-abe8-315bdd57b0ea)

Please consider fixing it! 🤓 🔧